tools/Exporter/Exporter.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>Microsoft.Health.Internal.Fhir.Exporter</AssemblyName>
+    <AssemblyName>Fhir.Exporter</AssemblyName>
```

Comment:
The same here.

Reply:
Keeping original because name is referenced.